README.md

```diff
@@ -14,27 +14,49 @@
 
 Cocotb can be installed by running `pip install cocotb`.
 
-## Quickstart
+## Usage
 
-    # Install pre-requisites (waveform viewer optional)
-    sudo yum install -y iverilog python-devel gtkwave
```

Comment:
Do we want to keep a mention of iverilog somewhere?

Reply:
Good question we could add something like:

### Prerequisites
- Python 3.5+ (with --embedded-shared)
- a compiler gcc/clang 
- an HDL simulator (example [Icarus Verilog](http://iverilog.icarus.com/)

Reply:
That seems pretty reasonable

Reply:
I'd remove the `--embedded-shared` part. This is a quickstart README, optimized for the standard user, who will get Python from a package manager. Instead, link to more detailed requirements (incl. compiler requirements) in the docs. 

Also link to a list of supported simulators, I get that question regularly.

Reply:
Maybe something like this:

In order to use cococtb Python interpreter, c/c++ compiler and HDL simulator is required to see more in [coumenation](https://cocotb.readthedocs.io/en/latest/quickstart.html#pre-requisites)

To not double the same information that can change.

Reply:
Agreed that we don't want the list in the readme. I think it's ok to name examples of such a simulator in the readme though.